README.md

```diff
@@ -37,6 +41,10 @@ React Native date & time picker component for iOS and Android
     - [`maximumDate` (`optional`)](#maximumdate-optional)
     - [`minimumDate` (`optional`)](#minimumdate-optional)
     - [`timeZoneOffsetInMinutes` (`optional`, `iOS only`)](#timezoneoffsetinminutes-optional-ios-only)
+    - [`timeZoneOffsetInSeconds` (`optional`, `Windows only`)](#timezoneoffsetinsecond-optional-windows-only)
```

Comment:
curious why have a separate property that is Windows only for this if we can piggyback on the iOS property?

Reply:
Filed #161

Reply:
Discussed separately, resolving.